service-front/module/Application/src/Controller/IndexController.php

```diff
@@ -46,6 +46,14 @@ public function startAction(): Response
 
         /** @var string $type */
         $type = $this->params()->fromQuery("personType");
+
```

Comment:
Can you delete rather than comment out if this is not needed any more (think it's related to our conversation re business rules)